README.md

```diff
@@ -178,3 +178,56 @@ bucket into a publicly browsable file tree. Useful if you want something
 like Apache's directory index for your software download repository.
 Plume release handles this as well, so it does not need to be run as part of
 the release process.
+
+## Cloud Provider Credentials
+Each platform reads the credentials it uses from different files.
+
+### qemu
+`qemu` is run locally and needs no credentials, but does need to be run as root.
+
+### aws
+`aws` uses the `~/.aws/credentials` file to store credentials. This can be populated manually:
+
+```
+[default]
+aws_access_key_id = ACCESS_KEY_ID_HERE
+aws_secret_access_key = SECRET_ACCESS_KEY_HERE
+```
+
+or can be populated using the `aws` command:
+```
+$ aws configure
+```
+
+To install the `aws` command from within the sdk, run `sudo emerge --ask awscli`.
+
+### gce
+`gce` uses the `~/.boto` file. This will be populated when the gce platform is first used.
```

Comment:
Let's consistently use `monospace` for platforms.

Reply:
Are you referring to the headings or the use of gce mid-sentence or both?

Reply:
The headings are a good point, but I meant the mid-sentence use.

Reply:
I think the headings are OK as is, the monospace grey background looks weird in headings to me.